libraries/standard/mqtt/src/mqtt_state.c

```diff
@@ -615,6 +621,10 @@ static uint16_t stateSelect( const MQTTContext_t * pMqttContext,
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @cond DOXYGEN_IGNORE
```

Comment:
Yes

Reply:
It warns function does not have documentation. I thought a bit more and I am going to add documentation in here since we don't want to the function to not appear at all.